jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -94,25 +97,16 @@ def process(self, input, *args, **kwargs):
             return result
 
         # fit sip approx., degree is chosen by best fit
+        sip_degree = range(1, _MAX_SIP_DEGREE) if self.sip_degree is None else self.sip_degree
+        sip_inv_degree = range(1, _MAX_SIP_DEGREE) if self.sip_inv_degree is None else self.sip_inv_degree
+        crpix = [result.meta.wcsinfo.crpix1, result.meta.wcsinfo.crpix2]
```

Comment:
It would be nice to not depend on crpix values as inputs.  We don't need to constrain the SIP to use the same ones as are done for the gwcs model.  Though clearly we could.  Is there a good reason to do this?

Reply:
What do you mean by "depend" and why it would be nice? I thought during one of the tag-ups you mentioned that for some instruments `crpix` is in a corner of the chip instead of being near the center of the bounding box. That was the motivation for https://github.com/spacetelescope/gwcs/pull/337 in the first place.

Reply:
Plus, there is no harm in not modifying CRPIX value already recorded in `wcsinfo`.

Reply:
That is, I see absolutely no benefit in moving/modifying `wcsinfo.crpix1,2`.

Reply:
Maybe if `CRPIXn` is not defined, default to the center?  So if `crpix = [None, None]`, set `crpix = None` and then do the fit?

Reply:
Done